make_prg/subcommands/update.py

```diff
@@ -0,0 +1,187 @@
+import multiprocessing
+import os
+from pathlib import Path
+from loguru import logger
+from make_prg.utils import io_utils, gfa
+from make_prg.update.denovo_variants import DenovoVariantsDB
+from make_prg.prg_builder import PrgBuilderZipDatabase, LeafNotFoundException
+from make_prg.utils.msa_aligner import MAFFT, MSAAligner
+from dataclasses import dataclass
+
+
+def register_parser(subparsers):
+    subparser_update_prg = subparsers.add_parser(
+        "update",
+        usage="make_prg update",
+        help="Update PRGs given new sequences.",
+    )
+
+    def check_if_is_update_file(argument: str):
+        argument = Path(argument)
+        is_zip_file = argument.suffix == ".zip"
+        is_update_DS = argument.suffix == ".update_DS"
+        if not is_zip_file and not is_update_DS:
+            subparser_update_prg.error(f"{argument} is not a update_DS nor a zip file.")
+        return argument
+    subparser_update_prg.add_argument(
+        "-u",
+        "--update_DS",
+        action="store",
+        type=lambda argument: check_if_is_update_file(argument),
+        required=True,
+        help=(
+            "Filepath to the update data structures (a *.update_DS.zip file created "
```

Comment:
I could also be a non-zipped file right?
```suggestion
            "Filepath to the update data structure(s) (*.update_DS.zip or *.update_DS) "
```

Reply:
Not right now, `make_prg` always creates a `*.update_DS.zip` file, even if you give only one MSA. In this case, this zip file will have a single file inside. This is just for uniformity when receiving a single MSA or a dir of MSAs in the input, `make_prg from_msa` will produce the same file type, that might be input into `make_prg update`. I have a slight preference to leave it like this, as if we create two different file types, we would need more code on output producing, input reading and more tests.

We do this though for the `.gfa` and `.bin` files (i.e. if it is a single MSA, a single `.gfa` and a single `.bin` files are created, but for multiple MSAs these are all bundled together in a single `.gfa.zip` and a single `.bin.zip` files), but we don't have to read these files back at any point later.

If you think this is a bad pattern for a CLI, then I am happy to change. Otherwise, if you have no preference, then I will keep it as is, as I have a slight preference for how it is now

Reply:
Oh sorry, I thought I saw some some code somewhere that was checking for both of those file extensions...

Fine to leave it as it is